assume/common/scenario_loader.py

```diff
@@ -378,13 +375,9 @@ async def load_scenario_folder_async(
     # if vre generation is provided, add them to the vre units
     # if we have RL strategy, add price forecast to unit_params
     def powerplant_callback(unit_name, unit_params):
-        # check if we have RL bidding strategy
-        if (
-            unit_params["bidding_strategies"]["energy"] == "rl_strategy"
-            and world.forecast_providers["EOM"].price_forecast_df is not None
-        ):
-            unit_params["price_forecast"] = world.price_forecast["mcp"]
-            unit_params["res_demand_forecast"] = world.price_forecast["residual_demand"]
```

Comment:
`unit_params["res_demand_forecast"] = world.price_forecast["residual_demand"]` went missing here? unneeded?

Reply:
this was needed for the learning agents. We will make a clean implementation with Kim once we Strat with it next week